asciidoc/courses/importing-data/modules/3-refactoring-imported-data/module.adoc

```diff
@@ -4,12 +4,12 @@
 [.transcript]
 == Module Overview
 
-If you use the Data Importer, you have learned that it imports all properties as string, integer, float, datetime, or boolean types.
+If you used the Data Importer, you have learned that it imports all properties as string, integer, float, datetime, or boolean types.
```

Comment:
If you _have_ used Data Importer, you _will_ have learned?

Reply:
adding "will" makes it more conversation, I tend to side on using less words. 🤷